bo/marktrechnung.go

```diff
@@ -0,0 +1,8 @@
+package bo
+
+// Marktrechnung ist eine Rechnung, die zwischen Marktteilnehmer n ausgetauscht wird.
+type Marktrechnung struct {
+	Rechnung
```

Comment:
Das daraus resultierende Json wird leider nicht korrekt sein. 

{
"Rechnung": {
    "rechnungsersteller": {},
    "rechnungsempfaenger": {}
},
"rechnungsersteller": {},
"rechnungsempfaenger": {}
}

So in der Art müsste es dann aussehen.

Reply:
Ne, durch das embedden geht das schon. Es wird im Json keine Unterstruktur aufgemacht und nur der Rechnungsersteller/Empfänger aus dem Marktteilnehmer wird serialisiert: c0f671659651000884fe4b9cfa84ec7e0fcd5895
Wir bekommen auch ausdrücklich keinen "Rechnung"-Key innerhalb des Marktrechnungsobjekts: 3bf206c4c7dd538770d1a30766b712ba74fbcdf2